meson.build

```diff
@@ -107,7 +110,7 @@ prefixdir = get_option('prefix')
 if not prefixdir.startswith('/')
         error('Prefix is not absolute: "@0@"'.format(prefixdir))
 endif
-if prefixdir != rootprefixdir and not prefixdir.startswith(rootprefixdir.strip('/') + '/')
+if prefixdir != rootprefixdir and not prefixdir.strip('/').startswith(rootprefixdir.strip('/') + '/')
```

Comment:
The change breaks when `rootprefixdir == '/'`.

This change is meaningless if `rootprefixdir == '/'`, so how about the following?
```
if prefixdir != rootprefixdir and rootprefixdir != '/' and not prefixdir.strip('/').startswith(rootprefixdir.strip('/') + '/')
```